project/src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,26 @@
+import {Route, Redirect} from 'react-router-dom';
+import {RouteProps} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../const';
+
+type PrivateRouteProps = RouteProps & {
+  render: () => JSX.Element;
+  authorizationStatus: AuthorizationStatus;
+}
+
+function PrivateRoute(props: PrivateRouteProps): JSX.Element {
+  const {exact, path, render, authorizationStatus} = props;
```

Comment:
можно сразу в параметрах делать деструктуризацию из объекта
```suggestion
function PrivateRoute({authorizationStatus, render, ...rest }: PrivateRouteProps): JSX.Element {
```